ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -1052,6 +1068,21 @@ export const ListSearch = fp.flow(
                     />
                   </span>
                 )}
+                {this.checkDrug && (
+                  <span style={{ float: 'right' }}>
+                    <span
+                      style={{ display: 'table-cell', paddingRight: '0.35rem' }}
+                    >
+                      Remove Brand Names
+                    </span>
+                    <InputSwitch
+                      checked={removeDrugBrand}
+                      disabled={loading}
```

Comment:
If we want to disable the switch when no brands are in the list, we can add something like this
```suggestion
                      disabled={
                        loading ||
                        !displayData?.some(
                          (row) => row.type === CriteriaType.BRAND
                        )
                      }
```